tfsdklog/sink.go

```diff
@@ -0,0 +1,96 @@
+package tfsdklog
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/terraform-plugin-log/internal/logging"
+)
+
+const (
+	envLog     = "TF_LOG"
+	envLogFile = "TF_LOG_PATH"
```

Comment:
Similar note here about (at least briefly) describing what each of these are/do.